dms/security/security.xml

```diff
@@ -115,7 +116,9 @@
         <field name="perm_create" eval="1" />
         <field name="perm_write" eval="0" />
         <field name="perm_unlink" eval="0" />
-        <field name="domain_force">[('permission_create', '=', user.id)]</field>
+        <field
+            name="domain_force"
+        >[('id', 'in', user._get_allowed_directory_ids("perm_create"))]</field>
```

Comment:
IMO this changes (and similar) is not necessary (as discussed in the `res.users` file changes).

Reply:
I rollbacked as what it was.